motoko/http_counter/src/main.mo

```diff
@@ -0,0 +1,150 @@
+import Nat "mo:base/Nat";
+import Text "mo:base/Text";
+import Array "mo:base/Array";
+import Option "mo:base/Option";
+import Prim "mo:prim";
```

Comment:
```suggestion
import Prim "mo:⛔";
```
(Our sad attempt to make it harder to access the compiler internal (and unstable) Prim library. 
The too easy `import Prim "mo:prim";` syntax will eventually be deprecated.